runbot_travis2docker/models/runbot_build.py

```diff
@@ -166,7 +164,7 @@ def job_30_run(self, cr, uid, build, lock_path, log_path):
     @custom_build
     def checkout(self, cr, uid, ids, context=None):
         """Save travis2docker output"""
-        to_be_skipped_ids = ids
+        to_be_skipped_ids = ids[:]
```

Comment:
Use a different pointer with the same values:
```bash
>>> my_list1 = ['a', 'b', 'c']
>>> my_list2 = my_list1
>>> id(my_list2) == id(my_list1)
True
>>> my_list2 = my_list1[:]
>>> id(my_list2) == id(my_list1)
False
```

This avoid change values from original pointer:
```bash
>>> my_list1 = ['a', 'b', 'c']
>>> my_list2 = my_list1
>>> my_list2.remove('a')
>>> my_list1
['b', 'c']  # Delete items from original variable because is a pointer. Bad!

my_list1 = ['a', 'b', 'c']
my_list2 = my_list1[:]
>>> my_list2.remove('a')
>>> my_list1
['a', 'b', 'c']  # Original values is not deleted. Good!
```